packages/core/src/components/Drawer/Drawer.styles.tsx

```diff
@@ -10,7 +11,9 @@ export const { staticClasses, useClasses } = createClasses("HvDrawer", {
     overflow: "auto",
     boxShadow: theme.colors.shadow,
   },
-  background: {},
+  background: {
```

Comment:
We should leave the `background` class empty since using `hexToRgbA` with a CSS var doesn't work and we already define the `background` inside the `Drawer.tsx` file.

<img width="516" alt="Screenshot 2023-11-02 at 10 33 55" src="https://github.com/lumada-design/hv-uikit-react/assets/43220251/0afd915e-1f3c-4da3-adfa-1e74ec830e67">

Reply:
Yes, these are leftovers. I've removed it.